src/components/FuelUsed/FuelForm.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+export default function FuelForm () {
+    return (
+        <div>
+            {/* Replace this placeholder route */}
+            <form action="/api/form" method="POST">
+                
+                {/* Add regex to validate answers once I know what the questions are. */}
+                <label htmlFor="api_name">Which fuel do you use?</label>
+                <select name="api_name" id="fuel_name">
```

Comment:
This should be tied to a state variable.

`const [fuelname, setFuelName] = useState<string>('')`